.github/workflows/neofs.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish to NeoFS
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  push-to-neofs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PATH_TO_FILES_DIR
+        run: |
+          echo "PATH_TO_FILES_DIR=output" >> $GITHUB_ENV
+
+      - name: Build Docker image
+        run: make image
+
+      - name: Get pdf file
+        run: make docker/pdf
+
+      - uses: actions/checkout@v4
+      - name: Publish to NeoFS
+        uses: nspcc-dev/gh-push-to-neofs@master
+        with:
+          NEOFS_WALLET: ${{ secrets.NEOFS_WALLET }}
+          NEOFS_WALLET_PASSWORD: ${{ secrets.NEOFS_WALLET_PASSWORD }}
+          NEOFS_NETWORK_DOMAIN: ${{ env.NEOFS_NETWORK_DOMAIN }}
+          NEOFS_HTTP_GATE: ${{ env.NEOFS_HTTP_GATE }}
+          STORE_OBJECTS_CID: ${{ env.STORE_OBJECTS_CID }}
+          PATH_TO_FILES_DIR: ${{ env.PATH_TO_FILES_DIR }}
```

Comment:
I think we should not have this `env` variable, we know exactly which files we want to upload, so action should receive this output directory and that's it.